test/parallel_api/ranges/std_ranges_sort.pass.cpp

```diff
@@ -24,8 +24,9 @@ main()
 
     auto sort_checker = TEST_PREPARE_CALLABLE(std::ranges::sort);
 
-    test_range_algo<0>{big_sz}(dpl_ranges::sort, sort_checker);
-    test_range_algo<1>{}(dpl_ranges::sort, sort_checker, std::ranges::less{});
+    test_range_algo<0, int, data_in, DevicePolicy>{big_sz}(dpl_ranges::sort, sort_checker);
+
+    test_range_algo<1, int, data_in, DeviceAndParPolicies>{medium_sz}(dpl_ranges::sort, sort_checker, std::ranges::less{});
```

Comment:
To me, it seems a little strange to sort a big size of 16M, a medium of 128K, and then a small of max 10 elements for tests 2-7.

I think we could increase the small (default) to something like 2k perhaps without any real effect on timing.

Reply:
Thanks, I've made a change to keep the coverage.

Reply:
Done